bugbug/models/annotate_ignore.py

```diff
@@ -79,7 +79,8 @@ def __init__(self, lemmatization: bool = False) -> None:
             ]
         )
 
-        self.clf = xgboost.XGBClassifier(n_jobs=utils.get_physical_cpu_count())
+        self.param = {"n_jobs": utils.get_physical_cpu_count()}
```

Comment:
I would call it `hyperparameter`:
```suggestion
        self.hyperparameter = {"n_jobs": utils.get_physical_cpu_count()}
```